### PR TITLE
HIP: AnonCreds Verifiable Data Registry

### PR DESCRIPTION
**Pull request by sumapnair-ibm**
**Description**:
This PR proposes a HIP to extend AnonCreds Verifiable Data Registry creation and maintenance in Hedera

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] hip-0000-anonCreds-verifiable-data-registry.md

### CODE REVIEW

File: hip-0000-anonCreds-verifiable-data-registry.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+hip: 0000
+title: AnonCreds Verifiable Data Registry
+author: [Sumabala P Nair](https://github.com/sumapnair-ibm), [Jack Hsu]()
```
Comment (mgarbs) [inline hip-0000-anonCreds-verifiable-data-registry.md:4]: Our validator checks for names in this form -> Firstname Lastname (@githubname or email)
```suggestion
author: Sumabala P Nair (@sumapnair-ibm), Jack Hsu (@jackgithubname or email)
```

Comment (sumapnair-ibm) [reply hip-0000-anonCreds-verifiable-data-registry.md]: @mgarbs When I try to sign off and commit the suggestion about the name, I  see the button is greyed out and github says 'Outdated suggestion cannot be applied' Please advise.

**Review by mgarbs — APPROVED** [review]

**Comment by fessmm** [comment]
@mgarbs this got merged in the wrong folder :)

it is located in:
hashgraph/hedera-improvement-proposal/hip-762.md

but it should be in:
hashgraph/hedera-improvement-proposal/HIP/hip-762.md

**Comment by sumapnair-ibm** [comment]
#1. What should be the structure of the identifiers for the Anoncred objects? 
I understand this is the structure that anoncreds generally follows. Do you know if there is a way for us to comply with the did:core spec recommendation? I have not had a conversation with anyone from the anoncreds world on this topic, but I think I remember a Hyperledger community  session where I think Stephen Curan mentioned it should be possible. Perhaps we should have a conversation with them? Thoughts?
#2. DID Linked Resources - I am in complete agreement with you. I think it is an unnecessary overhead for the Hedera architecture.
#3. What object operations will the Hedera Anoncreds method support? I agree we should be guided by what ACAPy comes up with; we should have an iterative roadmap with what to prioritize. I will also spend some time looking into the recent ACAPy work in the coming weeks.
#4. Store Revocation State or Deltas? Staying closer to the original implementation, with delta storage agrees with me as well. We should however explore and try to proactively mitigate any performance challenges that might arise out of the calculation.

**Pull request merged**